Mathlib/RingTheory/PowerSeries/Restricted.lean

```diff
@@ -0,0 +1,227 @@
+import Mathlib.Analysis.Normed.Group.Ultra
+import Mathlib.Analysis.RCLike.Basic
+import Mathlib.Data.Real.StarOrdered
+import Mathlib.RingTheory.PowerSeries.Basic
+import Mathlib.Analysis.Normed.Ring.Ultra
+
+variable (c : NNReal) (R : Type*) [NormedRing R]
```

Comment:
It's always useful to have a [module docstring](https://leanprover-community.github.io/contribute/style.html#module-docstrings) to give a brief description of the mathematical contents of your file + the main results and definitions!